Makefile

```diff
@@ -218,7 +205,7 @@ watch:
 # Download cache
 ################
 $(CACHE):
-	git clone --depth=1 "git@github.com:juju/juju-gui-downloadcache.git" $(CACHE)
+	git clone --depth=1 "https://github.com/juju/juju-gui-downloadcache.git" $(CACHE)
```

Comment:
Why this change? Not against it, just curious what difference it makes.

Reply:
I found that spinning up a bunch of lxc's and setting up ssh keys was a pain. So changing this made it easier :) It also turned out to take the same time as the ssh'd version